core/mondoo-okta-security.mql.yaml

```diff
@@ -8,7 +8,7 @@ policies:
     license: BUSL-1.1
     tags:
       mondoo.com/category: security
-      mondoo.com/platform: okta-org,saas
```

Comment:
not sure I understand why we are going to remove it

Reply:
These don't match any platforms we set. I can add them back in if we want to keep them for search. I'll just make sure they're on everything. Up to you.

Reply:
wouldn't that help us to search for saas, cloud etc?